plutus-core/plutus-core/test/Pretty/Golden/Readable/Examples/Data/exampleData.golden

```diff
@@ -1,7 +1,7 @@
-\(x : unit)
- (x : unit)
- (x : unit)
- (x : unit) ->
+\(x#0 : unit)
+ (x#0 : unit)
+ (x#0 : unit)
+ (x#0 : unit) ->
```

Comment:
I wonder what's that about, but we'll never know, because apparently it was me who added it.